### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by ichi-01**
Please merge the ones that correspond to multiple socat processes.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,23 +1,27 @@
...
-ARG DEF_LOCAL_PORT=80
-
-ENV REMOTE_PORT=$DEF_REMOTE_PORT
-ENV LOCAL_PORT=$DEF_LOCAL_PORT
-
-RUN apk update
-RUN apk add curl
-
-## By default container listens on $LOCAL_PORT (80)
 EXPOSE 80
```
Comment (RayMeibergen) [inline Dockerfile]: do you need this?

File: supervisor/create_conf.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+rm socat*.ini
```
Comment (RayMeibergen) [inline supervisor/create_conf.sh]: why do you have to delete ini first?

File: supervisor/create_conf.sh [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    LOCAL_PORT=${PARAMS[0]}
+    REMOTE_HOST=${PARAMS[1]}
+    REMOTE_PORT=${PARAMS[2]}
+
+    FILENAME=socat$number.ini
+    TITLE=[program:socat$number]
+    COMMAND=command="socat tcp-listen:$LOCAL_PORT,reuseaddr,fork tcp:$REMOTE_HOST:$REMOTE_PORT"
+    echo -e $TITLE > $FILENAME
+    echo -e $COMMAND >> $FILENAME
+    echo -e "stdout_logfile=/dev/stdout\nstdout_logfile_maxbytes=0\nstderr_logfile=/dev/stderr\nstderr_logfile_maxbytes=0" >> $FILENAME
```
Comment (RayMeibergen) [inline supervisor/create_conf.sh:29]:
```
FILENAME=/etc/supervisor.d/socat$number.ini
    echo -e $TITLE > $FILENAME
    echo -e $COMMAND >> $FILENAME
    echo -e "stdout_logfile=/dev/stdout\nstdout_logfile_maxbytes=0\nstderr_logfile=/dev/stderr\nstderr_logfile_maxbytes=0" >> $FILENAME
```

if this you don't have to below
`mv ./*.ini /etc/supervisor.d/`

**Review by RayMeibergen — COMMENTED** [review]
i made some comment to your code.

File: supervisor/create_conf.sh (hunk 3 above)
Comment (ichi-01) [reply supervisor/create_conf.sh:29]: Correspondence finished

File: supervisor/create_conf.sh (hunk 2 above)
Comment (ichi-01) [reply supervisor/create_conf.sh]: Correspondence finished

**Review by RayMeibergen — COMMENTED** [review]
OK

**Pull request closed without merging**